### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by josephworks**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,5 +9,5 @@ Today you can find, filter and review all sorts of open source up at https://ope
 At this time the site also continues to hold the official Microsoft Open Source Code of Conduct policy and
 FAQ.
 
-Feel free to email mailto:opensource@microsoft.com if you have any questions about this site or Microsoft GitHub
+Feel free to email [opensource@microsoft.com](mailto:opensource@microsoft.com) if you have any questions about this site or Microsoft GitHub
```
Comment (ralynn43) [inline README.md]: Thank you

Comment (josephworks) [reply README.md]: your welcome

Comment (ExE-Boss) [reply README.md]: It should work as just `opensource@microsoft.com` (opensource@microsoft.com) without the `[…](mailto:opensource@microsoft.com)` stuff.

Comment (ExE-Boss) [inline README.md]: It should work as just `opensource@microsoft.com` (opensource@microsoft.com) without the `[…](mailto:opensource@microsoft.com)` stuff.

**Comment by capfei** [comment]
I am closing this since it has been fixed in #155.

**Pull request closed without merging**